index.js

```diff
@@ -173,7 +173,14 @@ function compression(options) {
 
       // compression method
       var accept = accepts(req)
-      var method = accept.encoding(['gzip', 'deflate', 'identity'])
+      var method = null;
+
+      // If we support a custom encoding and a custom encoding was requested
+      if (opts.compressor) {
+        method = accept.encoding(Object.keys(opts.compressor));
```

Comment:
Ah, you're correct.  I think I'll need to implement turning off gzip or deflate then. Maybe:
```js
var opts = {
  compressor: {
    br: ...,
    gzip: null,
    deflate: null,
  }
};
```